eos-factory-reset

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -e
+
+cmd_name=$(basename $0)
+
+usage () {
+    cat <<EOF
+Usage: $cmd_name
+Reboot the machine and perform a factory reset.
+
+  -u               reset all users only
+  -a               make a full factory reset
```

Comment:
I guess this is for future-proofing but at present `-u` and `-a` are equivalent, and you have to specify one or the other. Seems a bit weird.